lib/ui/baseUI.js

```diff
@@ -21,7 +29,7 @@ var UI = module.exports = function (opt) {
   process.on('exit', this.onForceClose);
 
   // Propagate keypress events directly on the readline
-  process.stdin.addListener('keypress', this.onKeypress);
+  this._input.addListener('keypress', this.onKeypress);
```

Comment:
I'm fairly certain we can just call `this.rl.output`/`this.rl.input` without requiring us to keep the reference around.